text/3081-multi-part-examples-in-rustdoc/index.md

```diff
@@ -72,6 +72,10 @@ If `split-start` / `split-continue` / `split-end` blocks appear in an erroneous
 # Reference-level explanation
 [reference-level-explanation]: #reference-level-explanation
 
+UI Mockup:
+
+![UI mockup](./mockup.png)
```

Comment:
This has two different 'Run' buttons, one on each code block. Would that run the full example in both cases? If not, does it make sense to only add it to the second block?

Reply:
Now that I have looked into this RFC more, I am not certain about a very relevant point: those run buttons — where do I know them from? Where are they currently implemented? Is it in rustdoc or in mdBook? The only place I have certainly seen them is the standard library documentation.

This point is relevant because if they are not a rustdoc feature, then it would seem of no concern to this RFC and the mockup should not include such buttons.

Reply:
It is part of rustdoc. I think you need the `#![doc(html_playground_url)]` attribute for them to show up: https://github.com/rust-lang/rust/blob/4c55c8362de32be1530b2441c3e3a51e73edeb21/library/core/src/lib.rs#L58

Reply:
It's also implemented in mdBook, but as a button that looks (roughly) like this: ⏯️